functional_tests/scylla_operator/conftest.py

```diff
@@ -70,10 +70,13 @@ def fixture_tester() -> ScyllaOperatorFunctionalClusterTester:
     global TESTER  # pylint: disable=global-statement
     os.chdir(sct_abs_path())
     tester_inst = ScyllaOperatorFunctionalClusterTester()
+    TESTER = tester_inst  # putting tester global, so we can report skipped test (one with mark.skip)
     tester_inst.setUpClass()
     tester_inst.setUp()
-    TESTER = tester_inst  # putting tester global, so we can report skipped test (one with mark.skip)
     yield tester_inst
+    event = tester_inst._get_test_result_event()  # pylint: disable=protected-access
```

Comment:
If K8S cluster setup fails then this code won't be reached.
So, maybe you may want to combine this one with another your change: https://github.com/scylladb/scylla-cluster-tests/pull/4872 ?

Reply:
you have a point here, but it might be the other way around. and I'll bring the needed changes into this one.